cmd/snap-update-ns/mount-entry.c

```diff
@@ -197,7 +211,11 @@ void sc_save_mount_profile(const struct sc_mount_entry_list *list,
 	}
 }
 
-void sc_sort_mount_entry_list(struct sc_mount_entry_list *list)
+typedef int (*mount_entry_cmp_fn) (const struct sc_mount_entry **,
+				   const struct sc_mount_entry **);
+
+static void sc_sort_mount_entry_list_with(struct sc_mount_entry_list *list,
```

Comment:
Silly question - but why adding this helper ? Is there a followup branch that will use different sorting? If not, this feels like YAGNI to me.